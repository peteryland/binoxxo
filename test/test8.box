2o_o2x3x

oo2x_o_o
4o_x
_oo
o2x_oo3o
10o
4o3x
x4o_ox2x
_oo
5x_x3o
_x3x3x
