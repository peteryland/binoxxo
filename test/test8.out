__O_O__X___X
____________
OO__X_O_O___
____O_X_____
_OO_________
O__X_OO___O_
__________O_
____O___X___
X____O_OX__X
_OO_________
_____X_X___O
_X___X___X__

XOOXOOXXOOXX
XXOXOOXOXOXO
OOXOXXOXOXOX
OXXOOXXOOXXO
XOOXXOOXXOXO
OXOXXOOXOXOX
XOXOOXXOOXOX
OXOXOXOXXOXO
XOXOXOXOXOOX
XOOXXOXOOXOX
OXXOOXOXXOXO
OXXOXXOOXXOO
